internal/mail/mail.go

```diff
@@ -78,6 +78,16 @@ func SendIndividualMail(to []server.EmailRecipient, from server.EmailSender, cc
 		}
 	}
 
+	if len(tags) > 0 {
+		sendMessage.AddCategories(tags...)
+	}
+
+	if len(customArgs) > 0 {
+		for key, value := range customArgs {
+			sendMessage.SetCustomArg(key, value)
```

Comment:
apparently `SetCustomArg` is string/string
![CleanShot 2023-09-14 at 11 28 27](https://github.com/commitdev/zero-notification-service/assets/2908721/e35d2eb2-ae9e-4974-acaf-d4b0d78882f7)

Reply:
i guess u have to set each one by one, i wonder if the same key will override using their implementation 
but i guess not our problem